[Bug Report] Stopped working

### Bug Description

First of all, thank you for this project. Been working fine till recently.

Stopped reproducing music, and gives an error even after changing tokens.

### Steps to Reproduce

Just configure it like the manual says.
1 - Token and Visit Token in config file
2 - Launch app
3 - Play music
4 - error.

### Expected Result

To play the music, i guess.

### Debug Output

```
jmusicbot  | [15:31:28] [WARN] [SignatureCipherManager]: Transformed n parameter is the same as input, n function possibly short-circuited (in: tKdXEEjcFpfnw3y8p, out: tKdXEEjcFpfnw3y8p, player script: /s/player/f3d47b5a/player_ias.vflset/pt_PT/base.js, source version: 1.11.3)
jmusicbot  | [15:31:28] [WARN] [SignatureCipherManager]: Transformed n parameter is the same as input, n function possibly short-circuited (in: wPx-s13az5AWXg7Ru, out: wPx-s13az5AWXg7Ru, player script: /s/player/f3d47b5a/player_ias.vflset/pt_PT/base.js, source version: 1.11.3)
jmusicbot  | [15:31:28] [ERROR] [LocalAudioTrackExecutor]: Error in playback of VEJ8lpCQbyw
jmusicbot  | com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:45)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:121)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
jmusicbot  |    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
jmusicbot  |    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
jmusicbot  |    at java.base/java.lang.Thread.run(Thread.java:829)
jmusicbot  | Caused by: java.lang.RuntimeException: Not success status code: 403
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.validateStatusCode(PersistentHttpStream.java:88)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.attemptConnect(PersistentHttpStream.java:124)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.connect(PersistentHttpStream.java:106)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.internalRead(PersistentHttpStream.java:158)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.read(PersistentHttpStream.java:174)
jmusicbot  |    at java.base/java.io.DataInputStream.readByte(DataInputStream.java:270)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaEbmlReader.readEbmlInteger(MatroskaEbmlReader.java:39)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaFileReader.readNextElement(MatroskaFileReader.java:47)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.readFile(MatroskaStreamingFile.java:94)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.loadMatroskaFile(MatroskaAudioTrack.java:52)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.process(MatroskaAudioTrack.java:35)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25)
jmusicbot  |    at dev.lavalink.youtube.track.YoutubeAudioTrack.processStatic(YoutubeAudioTrack.java:159)
jmusicbot  |    at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:138)
jmusicbot  |    at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:79)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
jmusicbot  |    ... 4 common frames omitted
jmusicbot  |    Suppressed: dev.lavalink.youtube.ClientInformation: 
jmusicbot  |   yts.version: 1.11.3
jmusicbot  |   client.identifier: WEB_EMBEDDED_PLAYER
jmusicbot  |   client.options: ClientOptions{playback=true, playlistLoading=true, videoLoading=true, searching=true}
jmusicbot  | [15:31:28] [ERROR] [AudioHandler]: Track VEJ8lpCQbyw has failed to play
jmusicbot  | com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:45)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:121)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
jmusicbot  |    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
jmusicbot  |    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
jmusicbot  |    at java.base/java.lang.Thread.run(Thread.java:829)
jmusicbot  | Caused by: java.lang.RuntimeException: Not success status code: 403
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.validateStatusCode(PersistentHttpStream.java:88)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.attemptConnect(PersistentHttpStream.java:124)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.connect(PersistentHttpStream.java:106)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.internalRead(PersistentHttpStream.java:158)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.read(PersistentHttpStream.java:174)
jmusicbot  |    at java.base/java.io.DataInputStream.readByte(DataInputStream.java:270)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaEbmlReader.readEbmlInteger(MatroskaEbmlReader.java:39)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaFileReader.readNextElement(MatroskaFileReader.java:47)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.readFile(MatroskaStreamingFile.java:94)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.loadMatroskaFile(MatroskaAudioTrack.java:52)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.process(MatroskaAudioTrack.java:35)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25)
jmusicbot  |    at dev.lavalink.youtube.track.YoutubeAudioTrack.processStatic(YoutubeAudioTrack.java:159)
jmusicbot  |    at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:138)
jmusicbot  |    at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:79)
jmusicbot  |    at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
jmusicbot  |    ... 4 common frames omitted
jmusicbot  |    Suppressed: dev.lavalink.youtube.ClientInformation: 
jmusicbot  |   yts.version: 1.11.3
jmusicbot  |   client.identifier: WEB_EMBEDDED_PLAYER
jmusicbot  |   client.options: ClientOptions{playback=true, playlistLoading=true, videoLoading=true, searching=true}
```

### Additional Info

_No response_

### Checklist

- [x] I have looked for information about this [within the documentation](https://jmusicbot.com/)
- [x] I have searched for similar issues [on the issues page](https://github.com/jagrosh/MusicBot/issues?q=is%3Aissue)
- [x] I am using an official release from [the releases page](https://github.com/jagrosh/MusicBot/releases)

### JMusicBot Version

0.4.3.4

following. I have the same error. Token in config from docker image. Current release version.

```
java -jar JMusicBot-0.4.3.4.jar
[01:07:11] [WARN] [JMusicBot Version]: Switching to nogui mode. You can manually start in nogui mode by including the -Dnogui=true flag.
[01:07:11] [WARN] [JMusicBot Version]: There is a new version of JMusicBot available!
Current version: 0.4.3.4
New Version: 0.4.3

Please visit https://github.com/jagrosh/MusicBot/releases/latest to get the latest release.
[01:07:11] [INFO] [JMusicBot]: Loaded config from /home/discord/seville/config.txt
[01:07:11] [INFO] [Settings]: serversettings.json will be created in /home/discord/seville/serversettings.json
[01:07:11] [INFO] ytpotoken = MnS1aJ0GBMsm-LVQtKs5iPnd2hd6_09gGI9OxicgDnsmsxuD2syDJgAoWFsCBH1SVOF8UhwlwocZkb11Cmul4mh-XPvFWnKgh63vkhXW9rx-lMrSS8smSrpxs60CLzWfxWvva5hmSQxcziJk_plf3KzoXc42Bw==
[01:07:11] [INFO] ytvisitordata = CgstNHhDaXYwME5XbyjHrPe8BjIKCgJVUxIEGgAgJg%3D%3D
[01:07:11] [INFO] Config.txt succesfully updated!
[01:07:38] [INFO] [JDA]: Login Successful!
[01:07:38] [INFO] [WebSocketClient]: Connected to WebSocket
[01:07:38] [INFO] [JMusicBot]: JMusicBot
[01:07:39] [INFO] [JDA]: Finished Loading!
[01:08:18] [INFO] [YoutubeAccessTokenTracker]: Updating YouTube visitor id succeeded, new one is CgtGaDFiSmFkaWg3WSjyrPe8BjIKCgJVUxIEGgAgPzoMCAEgwbj8naHO9c5n, next update will be after 600 seconds.
[01:08:19] [WARN] [SignatureCipherManager]: Transformed n parameter is the same as input, n function possibly short-circuited (in: GfTi7eQxFXE57GbxU, out: GfTi7eQxFXE57GbxU, player script: /s/player/f3d47b5a/player_ias.vflset/en_US/base.js, source version: 1.11.3)
[01:08:20] [WARN] [SignatureCipherManager]: Transformed n parameter is the same as input, n function possibly short-circuited (in: 29IBioH45YLT3wmwj, out: 29IBioH45YLT3wmwj, player script: /s/player/f3d47b5a/player_ias.vflset/en_US/base.js, source version: 1.11.3)
[01:08:20] [ERROR] [LocalAudioTrackExecutor]: Error in playback of xgmWHBSZAUg
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:45)
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:121)
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.RuntimeException: Not success status code: 403
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.validateStatusCode(PersistentHttpStream.java:88)
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.attemptConnect(PersistentHttpStream.java:124)
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.connect(PersistentHttpStream.java:106)
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.internalRead(PersistentHttpStream.java:158)
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.read(PersistentHttpStream.java:174)
        at java.base/java.io.DataInputStream.readUnsignedByte(DataInputStream.java:295)
        at java.base/java.io.DataInputStream.readByte(DataInputStream.java:275)
        at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaEbmlReader.readEbmlInteger(MatroskaEbmlReader.java:39)
        at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaFileReader.readNextElement(MatroskaFileReader.java:47)
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.readFile(MatroskaStreamingFile.java:94)
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.loadMatroskaFile(MatroskaAudioTrack.java:52)
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.process(MatroskaAudioTrack.java:35)
        at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25)
        at dev.lavalink.youtube.track.YoutubeAudioTrack.processStatic(YoutubeAudioTrack.java:159)
        at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:138)
        at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:79)
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
        ... 4 common frames omitted
        Suppressed: dev.lavalink.youtube.ClientInformation:
  yts.version: 1.11.3
  client.identifier: WEB_EMBEDDED_PLAYER
  client.options: ClientOptions{playback=true, playlistLoading=true, videoLoading=true, searching=true}
[01:08:20] [ERROR] [AudioHandler]: Track xgmWHBSZAUg has failed to play
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:45)
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:121)
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.RuntimeException: Not success status code: 403
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.validateStatusCode(PersistentHttpStream.java:88)
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.attemptConnect(PersistentHttpStream.java:124)
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.connect(PersistentHttpStream.java:106)
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.internalRead(PersistentHttpStream.java:158)
        at com.sedmelluq.discord.lavaplayer.tools.io.PersistentHttpStream.read(PersistentHttpStream.java:174)
        at java.base/java.io.DataInputStream.readUnsignedByte(DataInputStream.java:295)
        at java.base/java.io.DataInputStream.readByte(DataInputStream.java:275)
        at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaEbmlReader.readEbmlInteger(MatroskaEbmlReader.java:39)
        at com.sedmelluq.discord.lavaplayer.container.matroska.format.MatroskaFileReader.readNextElement(MatroskaFileReader.java:47)
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaStreamingFile.readFile(MatroskaStreamingFile.java:94)
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.loadMatroskaFile(MatroskaAudioTrack.java:52)
        at com.sedmelluq.discord.lavaplayer.container.matroska.MatroskaAudioTrack.process(MatroskaAudioTrack.java:35)
        at com.sedmelluq.discord.lavaplayer.track.DelegatedAudioTrack.processDelegate(DelegatedAudioTrack.java:25)
        at dev.lavalink.youtube.track.YoutubeAudioTrack.processStatic(YoutubeAudioTrack.java:159)
        at dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:138)
        at dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:79)
        at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109)
        ... 4 common frames omitted
        Suppressed: dev.lavalink.youtube.ClientInformation:
  yts.version: 1.11.3
  client.identifier: WEB_EMBEDDED_PLAYER
  client.options: ClientOptions{playback=true, playlistLoading=true, videoLoading=true, searching=true}

```

--- a/.github/workflows/build-and-test.yml
+++ b/.github/workflows/build-and-test.yml
@@ -11,9 +11,9 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v4
         with:
           java-version: '11'
           distribution: 'adopt'

--- a/.github/workflows/make-release.yml
+++ b/.github/workflows/make-release.yml
@@ -15,11 +15,13 @@ jobs:
   build_jar:
     name: Build Jar
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v4
         with:
           java-version: '11'
           distribution: 'adopt'
@@ -34,7 +36,7 @@ jobs:
       - name: Rename jar
         run: mv target/*-All.jar JMusicBot-${{ github.event.inputs.version_number }}.jar
       - name: Upload jar
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: jar
           path: JMusicBot-${{ github.event.inputs.version_number }}.jar
@@ -45,26 +47,17 @@ jobs:
     needs: build_jar
     steps:
       - name: Download a Build Artifact
-        uses: actions/download-artifact@v3.0.0
+        uses: actions/download-artifact@v4
         with:
           name: jar
           path: .
-      - name: Show Artifacts
-        run: ls -R
       - name: Create Release
         uses: ncipollo/release-action@v1
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
           tag: ${{ github.event.inputs.version_number }}
-          name: Version ${{ github.event.inputs.version_number }}
+          name: ${{ github.event.inputs.version_number }}
           draft: true
           prerelease: false
-          artifacts: "*.jar"
+          artifacts: "JMusicBot-${{ github.event.inputs.version_number }}.jar"
           body: |
             ${{ github.event.inputs.info }}
-            
-            ---
-            ### Setup
-            https://jmusicbot.com/setup  
-            https://jmusicbot.com/config  
-            # Download: [JMusicBot-${{ github.event.inputs.version_number }}.jar](https://github.com/jagrosh/MusicBot/releases/download/${{ github.event.inputs.version_number }}/JMusicBot-${{ github.event.inputs.version_number }}.jar)

--- a/pom.xml
+++ b/pom.xml
@@ -64,7 +64,7 @@
         <dependency>
             <groupId>dev.lavalink.youtube</groupId>
             <artifactId>common</artifactId>
-            <version>1.11.3</version>
+            <version>1.11.4</version>
         </dependency>
         <dependency>
             <groupId>com.github.jagrosh</groupId>

